blocks/columns/columns.css

```diff
@@ -307,6 +307,10 @@ div.img-col > video {
   color: var(--secondary);
 }
 
+.columns  > div > div p.button-container a {
```

Comment:
Could we maybe move this margin to the button-container itself instead of each button? Cause we can have cases such as this where the 2 buttons won't fit on a single line and this will mean the buttons will get spacing added between them, here's the link https://awards-page--sunstar-foundation--hlxsites.hlx.live/about-us/history
![Screenshot 2024-01-11 at 09 51 52](https://github.com/hlxsites/sunstar-foundation/assets/44490413/6dd34120-b1e0-48fe-b6db-b3f0008624e9)
But I'm not entirely sure; why do we need this? Because this will also change the spacing on mobile when the flex direction changes and we have columns with buttons
![Screenshot 2024-01-11 at 09 56 19](https://github.com/hlxsites/sunstar-foundation/assets/44490413/3f3e9e70-0f56-48da-b482-8d21105492d2)